lib/milc_interface.cpp

```diff
@@ -51,7 +51,7 @@ static const int num_colors = sizeof(colors)/sizeof(uint32_t);
 
```

Comment:
I don't think this file should be changed at all

Reply:
it doesn't look. like it has been.

Reply:
Ah this is just replacing gridDim with commGridDim, no functionality changes.

Reply:
This may end up being a CMake level fix once CMake has full Language Support for HIP